game/scripts/vscripts/items/farming/greater_arcane_boots.lua

```diff
@@ -64,6 +64,26 @@ function item_greater_arcane_boots:GetIntrinsicModifierNames()
   }
 end
 
+
+local function GoldTickDown(hero)
```

Comment:
Code for handling the GPM should go into `modifier_passive_gpm.lua`. Likely you'll want to be modifying the `OnIntervalThink` function. Possibly making the reduction interval based on the GPM ticks would be easier.

Reply:
Did you forget to commit it? This branch has no changes to modifier_passive_gpm.

Reply:
I don't think so... maybe it's because I modified it in a previous pill request and haven't changed it since?

Reply:
Well, I dunno how or why, but I'm certain that this branch does not have any changes to modifier_passive_gpm.lua.